src/PanelTraits/Create.php

```diff
@@ -43,6 +51,24 @@ public function getCreateFields()
     }
 
     /**
+     * Get all fields that should be set after creation (after_create key set on field).
+     *
+     * @return [array] The fields with after_create key set.
+     */
+    public function getAfterCreateFields()
+    {
+        $fields = $this->create_fields;
+        $relationFields = [];
+
+        foreach ($fields as $field) {
```

Comment:
I wonder if `array_filter` would be faster.

Reply:
Or even better, use and return a collection :

```php
public function getAfterCreateFields()
{
    $relationFields = collect($this->create_fields)->filter(function($field) {
        return isset($field['after_create'];
    });

    return $relationFields;
}
```

Then replace :
```php
$after_create = array_pluck($this->getAfterCreateFields(), 'name');
```
... with :
```php
$after_create = $this->getAfterCreateFields()->pluck('name')->all();
```